yarn.lock

```diff
@@ -10412,6 +10412,15 @@ postcss@^8.2.15, postcss@^8.3.11, postcss@^8.4.5:
     picocolors "^1.0.0"
     source-map-js "^1.0.1"
 
+postcss@^8.4.7:
```

Comment:
looks like this could be deduped with the above?

Reply:
Thanks for catching this, should be fixed now.